modules/currency.ts

```diff
@@ -205,6 +215,16 @@ function responsesReadyHook(next, ready) {
   next(ready.then(() => responseReady.promise));
 }
 
+declare module '../src/bidfactory' {
+    interface BaseBid {
+        /**
```

Comment:
Same here. This type already exists in the `bidfactory.ts` and can be extended there.